src/assert_yaml_snapshot.rs

```diff
@@ -0,0 +1,18 @@
+/// Platform-independent snapshot assertion
+#[macro_export]
+macro_rules! assert_yaml_snapshot {
+    ( $($args: expr),+) => {
+        let line_sep = if cfg!(windows) { "\r\n" } else { "\n" };
+
+        // adjust snapshot file for platform
+        let mut settings = insta::Settings::clone_current();
+        settings.add_redaction("[].fix.content", insta::dynamic_redaction(move |value, _path| {
+            // CRLF/LF => <NEWLINE>
+            // TODO: do not replace escaped!
+            value.as_str().unwrap().replace(line_sep, "<NEWLINE>")
```

Comment:
For more readable output, we could _split_ on newlines instead, yielding a list.

```suggestion
            insta::internals::Content::Seq(
                value.as_str().unwrap().split(line_sep).map(|line| line.into()).collect()
            )
```

Reply:
Much better!